build/windows/nanoserver/Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM microsoft/nanoserver
 
+USER ContainerAdministrator
```

Comment:
Quick question, will this statement cause any issue with current windows images or 1709 ?

Reply:
No. I just quickly tested that ```docker run -t --rm -u ContainerAdministrator microsoft/nanoserver:1709 cmd /c echo %USERNAME%``` command works just fine.

Reply:
And on microsoft/nanoserver:sac2016 it actually does not change anything as ContainerAdministrator is default setting on that version (but also does not break anything).

Reply:
Yes, still fine for 1709 and 1803 windows images.

Reply:
Great, thanks guys